CMakeLists.txt

```diff
@@ -14,6 +10,10 @@ add_definitions(
 	/D_CRT_SECURE_NO_WARNINGS # disable warnings about old libc functions
 )
 set(CMAKE_CXX_FLAGS "/GR- ${CMAKE_CXX_FLAGS}") # turn off C++ RTTI
+set(CMAKE_C_FLAGS_RELEASE "/GL /Gw")
+set(CMAKE_CXX_FLAGS_RELEASE "/GL /Gw")
```

Comment:
The original flag override adds `/MT` so the redistributable runtime is statically linked to the `.dll`.

Should we keep `/MT` or you want to package the redistributable in the installer instead?

Reply:
oops, my mistake here.
I'll keep the static runtime library via a cmake variable.